src/ImageSharp/Formats/Heif/Heif4CharCode.cs

```diff
@@ -0,0 +1,261 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+// <auto-generated />
+
+using System.CodeDom.Compiler;
+
+namespace SixLabors.ImageSharp.Formats.Heif;
+
+/// <summary>
+/// Supported 4 character codes for use in HEIF images.
+/// </summary>
+[GeneratedCode("T4", "")]
+public enum Heif4CharCode : uint
+{
+    /// <summary>
+    /// File Type.
+    /// </summary>
+    ftyp = 0x66747970U,
```

Comment:
These will all need to use the correct casing.

Reply:
OK, went for automaticaly generated PascalCasing for now, if this needs to be nicer, can be done with third element.